config/samples/rabbitmq.com_v1alpha2_schemareplication.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: rabbitmq.com/v1alpha2
+kind: SchemaReplication
+metadata:
+  name: schemareplication-sample
+spec:
+  # Add fields here
+  foo: bar
```

Comment:
Did we mean to leave this as the sample YAML

Reply:
Actual example lives in `docs/example`. Manifests `config/samples` are generated by kubebuilder and not actually valid. I can create a separate PR to get rid of all of them.